music21/musicxml/m21ToXml.py

```diff
@@ -3870,14 +3870,13 @@ def noteToNotations(self, n, notFirstNoteOfChord=False, chordParent=None):
 
         notations = []
 
-        if notFirstNoteOfChord is False:
-            # only apply expressions and articulations
-            # to notes or the first note of a chord...
-            chordOrNote = n
-            if chordParent is not None:
-                # get expressions from first note of chord
-                chordOrNote = chordParent
+        chordOrNote = n
+        if chordParent is not None:
+            # get expressions from first note of chord
+            chordOrNote = chordParent
 
+        # only apply expressions to notes or the first note of a chord...
+        if pitchNumber == 0:
```

Comment:
I still prefer setting up a separate variable that makes more clear what "pitchNumber == 0" means:
`if isSingleNoteOrFirstInChord:`  even a wordy expression like this is helpful to show what is being tested.